### ISSUE TITLE
Don't draw entrance when also tagged with housenumber or unit

### ISSUE THREAD

**Issue by RedAuburn** [organicmaps/organicmaps#8522]
The door below the housenumber looks bad, we should only show the housenumber.

![IMG_20240622_134759_660.jpg](https://github.com/organicmaps/organicmaps/assets/26939824/dd541d07-bacf-43dd-abc3-5ae2a732bef6)

**Comment by biodranik** [comment]
Agree, a door can be implied from a number on a building's border. Only the icon can be removed, the selection should still show the "Entrance" translation.
